pkg/backends/datadog/datadog.go

```diff
@@ -258,28 +258,44 @@ func NewClientFromViper(v *viper.Viper) (gostatsd.Backend, error) {
 	dd.SetDefault("api_endpoint", apiURL)
 	dd.SetDefault("metrics_per_batch", defaultMetricsPerBatch)
 	dd.SetDefault("compress_payload", true)
-	dd.SetDefault("dual_stack", false)
+	dd.SetDefault("ipv4", true)
+	dd.SetDefault("ipv6", true)
 	dd.SetDefault("client_timeout", defaultClientTimeout)
 	dd.SetDefault("max_request_elapsed_time", defaultMaxRequestElapsedTime)
+
+	ipv4 := dd.GetBool("ipv4")
+	ipv6 := dd.GetBool("ipv6")
+	var network string
+	if ipv4 && ipv6 {
+		network = "tcp"
+	} else if ipv4 {
+		network = "tcp4"
+	} else if ipv6 {
+		network = "tcp6"
+	}
```

Comment:
Should probably return an error if both are false?

Reply:
I had that initially, then moved the validation logic to be with the rest of its friends below.